.gitignore

```diff
@@ -20,9 +20,7 @@ build
 # other
 eclipse
 run
-forge-1.12-14.21.1.2443-changelog.txt
-Paulscode SoundSystem CodecIBXM License.txt
-Paulscode IBXM Library License.txt
-MinecraftForge-Credits.txt
-CREDITS-fml.txt
-deps/
```

Comment:
Making these changes has added a lot of extra files to the diff.  Recommend undoing these changes and just adding `*.jar` instead

Reply:
Fixed